content/source/docs/enterprise/api/admin/users.html.md

```diff
@@ -473,23 +479,43 @@ Because of the requirement to provide a valid admin user session cookie in order
 
 Status  | Response                  | Reason
 --------|---------------------------|----------
-[204][] | Empty body                | Successfully impersonated the user
+[204][] | Empty body                | Successfully impersonated the user.
+[400][] | [JSON API error object][] | A reason for impersonation is required.
 [403][] | [JSON API error object][] | Client is already impersonating another user.
+[403][] | [JSON API error object][] | User is a Service Account which cannot be impersonated.
```

Comment:
Admins are also not able to be impersonated!

Reply:
That's not the case anymore; there was a change a while back (at my request) that removed that restriction.

Reply:
I was gonna say, I don't see that in the code

Reply:
Oh! My mistake, then